docs/content/users/install/ddev-installation.md

```diff
@@ -149,7 +149,7 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     You can install DDEV on Windows three ways:
 
-    1. [Using WSL2 with Docker inside](#wsl2-docker-ce-inside-install-script) **Recommended, best performance, most reliable**
+    1. [Using WSL2 with Docker Engine](#wsl2-docker-engine-install-script) **Recommended, best performance, most reliable**
```

Comment:
"Docker inside" is in fact very poor usage here, and we can do better. "Docker Engine" isn't great, because it allows confusion with Docker Desktop, which also has "Docker Engine" in it.